cypress/integration/select_spec.js

```diff
@@ -10,89 +11,18 @@ describe('New Select', function() {
+
     viewport.forEach(view => {
       before(function() {
         cy.viewport(view);
       });
-      beforeEach(function() {
-        cy.reload();
-      });
-      it('Should display 3 default values ' + view, function() {
-        cy
-          .get(selector.singleSelectDefaultValues)
-          .should('have.length', 3)
-          .get(selector.singleSelectFirstValue)
-          .should('contain', 'Ocean');
-      });
-      it('Should clear the default value ' + view, function() {
-        cy
-          .get(selector.clearValues)
-          .should('have.length', 1)
-          .click()
-          .each(function(element) {
-            expect(element).to.not.be.visible; // eslint-disable-line no-unused-expressions
-          });
-      });
-      it('Should expand the menu when click on the arrow ' + view, function() {
-        cy
-          .get(selector.toggleMenus)
-          .should('have.length', 8)
-          .get(selector.menuSingle)
-          .should('not.exist')
-          .get(selector.toggleMenuSingle)
-          .click()
-          .get(selector.menuSingle)
-          .should('exist')
-          .get(selector.menuSingle)
-          .should('be.visible')
-          .get(selector.menuOption)
-          .contains('Green')
-          .click()
-          .get(selector.singleInputValue)
-          .should('contain', 'Green');
-      });
-      it(
-        'Should expand the menu when enter a value and filter ' + view,
-        function() {
-          cy
-            .get(selector.singleSelectGroupedInput)
-            .click({ force: true })
-            .type('Stra', { force: true })
-            .type('{enter}', { force: true })
-            .type('{enter}', { force: true })
-            .get(selector.singleGroupedInputValue)
-            .should('contain', 'Strawberry');
-        }
-      );
-      it('Should return no options ' + view, function() {
-        cy
-          .get(selector.singleSelectGroupedInput)
-          .click({ force: true })
-          .type('/', { force: true })
-          .get(selector.noOptionsValue)
-          .should('contain', 'No options');
-      });
-      it('Should be disabled once disabled is checked ' + view, function() {
-        cy
-          .get(selector.disabledCheckbox)
-          .should('be.visible')
-          .get(selector.disabledCheckbox)
-          .click()
-          .get(selector.toggleMenuSingle)
-          .click({ force: true })
-          .get(selector.singleSelectSingleInput)
-          .should('be.disabled');
-      });
-      it('Should display group in the menu ' + view, function() {
-        cy
-          .get(selector.toggleMenuGrouped)
-          .click()
-          .get(selector.menuGrouped)
-          .should('be.visible')
-          .get(selector.groupColor)
-          .should('be.visible');
-      });
-      it(
+
+      // This test seems to fail when cypress tab is focused.
+      // Also, manual testing does not confirm the desired behavior
+      it.skip(
```

Comment:
It's not entirely clear to me what this test is trying to assert, and it doesn't seem to work properly.  I'd like to remove it, if that's okay with the maintainers.  Otherwise, we should probably fix it so that it passes when the cypress browser is focused during the tests.

Reply:
I believe the intention of this is to ensure that on mobile the menu isn't triggered on a drag event, if you can fix it within this PR that would be great, if not we can leave it skipped for now and come back to it in a subsequent body of work

Reply:
Yeah to be honest I'm really not sure how to get it working, and would love if we could leave it skipped for now and allow someone with more knowledge of testing mobile behavior in cypress to step in and get it working correctly.